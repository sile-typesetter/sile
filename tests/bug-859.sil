\begin[papersize=a6]{document}
\nofolios
\script[src=packages/leaders]
\set[parameter=typesetter.parfillskip,value=0pt]
\noindent\font[size=20pt]{A\dotfill{}A}\medskip
\noindent\font[size=10pt]{DEF\dotfill{}10}\medskip
\noindent\font[size=10pt]{GHI\dotfill{}100}\medskip
\noindent\font[size=10pt]{GHI\hfill{}100}\medskip
\noindent\font[size=60pt]{JKL\dotfill{}1000}\medskip
\noindent\font[size=52pt]{JKL\dotfill{}1000}\medskip
\end{document}
