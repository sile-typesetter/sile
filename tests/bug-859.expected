Set paper size 	297.6377985	419.5275636
Begin page
Mx 	14.8819
My 	33.8377
Set font 	Gentium Plus;20;400;;normal;;LTR
T	36 w=11.9043	(A)
Mx 	34.8946
Mx 	39.8946
T	17 w=4.5801	(.)
Mx 	49.4747
Mx 	54.4747
T	17 w=4.5801	(.)
Mx 	64.0547
Mx 	69.0547
T	17 w=4.5801	(.)
Mx 	78.6348
Mx 	83.6348
T	17 w=4.5801	(.)
Mx 	93.2149
Mx 	98.2149
T	17 w=4.5801	(.)
Mx 	107.7950
Mx 	112.7950
T	17 w=4.5801	(.)
Mx 	122.3750
Mx 	127.3750
T	17 w=4.5801	(.)
Mx 	136.9551
Mx 	141.9551
T	17 w=4.5801	(.)
Mx 	151.5352
Mx 	156.5352
T	17 w=4.5801	(.)
Mx 	166.1153
Mx 	171.1153
T	17 w=4.5801	(.)
Mx 	180.6954
Mx 	185.6954
T	17 w=4.5801	(.)
Mx 	195.2754
Mx 	200.2754
T	17 w=4.5801	(.)
Mx 	209.8555
Mx 	214.8555
T	17 w=4.5801	(.)
Mx 	224.4356
Mx 	229.4356
T	17 w=4.5801	(.)
Mx 	239.0157
Mx 	244.0157
T	17 w=4.5801	(.)
Mx 	253.5958
Mx 	258.5958
T	17 w=4.5801	(.)
Mx 	270.8516
T	36 w=11.9043	(A)
Mx 	14.8819
My 	51.8377
Set font 	Gentium Plus;10;400;;normal;;LTR
T	39 40 41 w=15.6689	(DEF)
Mx 	34.8946
Mx 	37.3946
T	17 w=2.2900	(.)
Mx 	42.1846
Mx 	44.6846
T	17 w=2.2900	(.)
Mx 	49.4747
Mx 	51.9747
T	17 w=2.2900	(.)
Mx 	56.7647
Mx 	59.2647
T	17 w=2.2900	(.)
Mx 	64.0547
Mx 	66.5547
T	17 w=2.2900	(.)
Mx 	71.3448
Mx 	73.8448
T	17 w=2.2900	(.)
Mx 	78.6348
Mx 	81.1348
T	17 w=2.2900	(.)
Mx 	85.9249
Mx 	88.4249
T	17 w=2.2900	(.)
Mx 	93.2149
Mx 	95.7149
T	17 w=2.2900	(.)
Mx 	100.5049
Mx 	103.0049
T	17 w=2.2900	(.)
Mx 	107.7950
Mx 	110.2950
T	17 w=2.2900	(.)
Mx 	115.0850
Mx 	117.5850
T	17 w=2.2900	(.)
Mx 	122.3750
Mx 	124.8750
T	17 w=2.2900	(.)
Mx 	129.6651
Mx 	132.1651
T	17 w=2.2900	(.)
Mx 	136.9551
Mx 	139.4551
T	17 w=2.2900	(.)
Mx 	144.2452
Mx 	146.7452
T	17 w=2.2900	(.)
Mx 	151.5352
Mx 	154.0352
T	17 w=2.2900	(.)
Mx 	158.8252
Mx 	161.3252
T	17 w=2.2900	(.)
Mx 	166.1153
Mx 	168.6153
T	17 w=2.2900	(.)
Mx 	173.4053
Mx 	175.9053
T	17 w=2.2900	(.)
Mx 	180.6954
Mx 	183.1954
T	17 w=2.2900	(.)
Mx 	187.9854
Mx 	190.4854
T	17 w=2.2900	(.)
Mx 	195.2754
Mx 	197.7754
T	17 w=2.2900	(.)
Mx 	202.5655
Mx 	205.0655
T	17 w=2.2900	(.)
Mx 	209.8555
Mx 	212.3555
T	17 w=2.2900	(.)
Mx 	217.1456
Mx 	219.6456
T	17 w=2.2900	(.)
Mx 	224.4356
Mx 	226.9356
T	17 w=2.2900	(.)
Mx 	231.7256
Mx 	234.2256
T	17 w=2.2900	(.)
Mx 	239.0157
Mx 	241.5157
T	17 w=2.2900	(.)
Mx 	246.3057
Mx 	248.8057
T	17 w=2.2900	(.)
Mx 	253.5958
Mx 	256.0958
T	17 w=2.2900	(.)
Mx 	260.8858
Mx 	263.3858
T	17 w=2.2900	(.)
Mx 	273.3711
T	20 19 w=9.3848	(10)
Mx 	14.8819
My 	69.8377
T	42 43 44 w=15.3467	(GHI)
Mx 	34.8946
Mx 	37.3946
T	17 w=2.2900	(.)
Mx 	42.1846
Mx 	44.6846
T	17 w=2.2900	(.)
Mx 	49.4747
Mx 	51.9747
T	17 w=2.2900	(.)
Mx 	56.7647
Mx 	59.2647
T	17 w=2.2900	(.)
Mx 	64.0547
Mx 	66.5547
T	17 w=2.2900	(.)
Mx 	71.3448
Mx 	73.8448
T	17 w=2.2900	(.)
Mx 	78.6348
Mx 	81.1348
T	17 w=2.2900	(.)
Mx 	85.9249
Mx 	88.4249
T	17 w=2.2900	(.)
Mx 	93.2149
Mx 	95.7149
T	17 w=2.2900	(.)
Mx 	100.5049
Mx 	103.0049
T	17 w=2.2900	(.)
Mx 	107.7950
Mx 	110.2950
T	17 w=2.2900	(.)
Mx 	115.0850
Mx 	117.5850
T	17 w=2.2900	(.)
Mx 	122.3750
Mx 	124.8750
T	17 w=2.2900	(.)
Mx 	129.6651
Mx 	132.1651
T	17 w=2.2900	(.)
Mx 	136.9551
Mx 	139.4551
T	17 w=2.2900	(.)
Mx 	144.2452
Mx 	146.7452
T	17 w=2.2900	(.)
Mx 	151.5352
Mx 	154.0352
T	17 w=2.2900	(.)
Mx 	158.8252
Mx 	161.3252
T	17 w=2.2900	(.)
Mx 	166.1153
Mx 	168.6153
T	17 w=2.2900	(.)
Mx 	173.4053
Mx 	175.9053
T	17 w=2.2900	(.)
Mx 	180.6954
Mx 	183.1954
T	17 w=2.2900	(.)
Mx 	187.9854
Mx 	190.4854
T	17 w=2.2900	(.)
Mx 	195.2754
Mx 	197.7754
T	17 w=2.2900	(.)
Mx 	202.5655
Mx 	205.0655
T	17 w=2.2900	(.)
Mx 	209.8555
Mx 	212.3555
T	17 w=2.2900	(.)
Mx 	217.1456
Mx 	219.6456
T	17 w=2.2900	(.)
Mx 	224.4356
Mx 	226.9356
T	17 w=2.2900	(.)
Mx 	231.7256
Mx 	234.2256
T	17 w=2.2900	(.)
Mx 	239.0157
Mx 	241.5157
T	17 w=2.2900	(.)
Mx 	246.3057
Mx 	248.8057
T	17 w=2.2900	(.)
Mx 	253.5958
Mx 	256.0958
T	17 w=2.2900	(.)
Mx 	260.8858
Mx 	263.3858
T	17 w=2.2900	(.)
Mx 	268.6788
T	20 19 19 w=14.0771	(100)
Mx 	14.8819
My 	87.8377
T	42 43 44 w=15.3467	(GHI)
Mx 	268.6788
T	20 19 19 w=14.0771	(100)
Mx 	14.8819
My 	131.8983
Set font 	Gentium Plus;60;400;;normal;;LTR
T	45 46 47 w=81.5625	(JKL)
Mx 	107.7950
Mx 	122.7950
T	17 w=13.7402	(.)
Mx 	170.1387
T	20 19 19 19 w=112.6172	(1000)
Mx 	14.8819
My 	182.0233
Set font 	Gentium Plus;52;400;;normal;;LTR
T	45 46 47 w=70.6875	(JKL)
Mx 	93.2149
Mx 	106.2149
T	17 w=11.9082	(.)
Mx 	131.1231
Mx 	144.1231
T	17 w=11.9082	(.)
Mx 	185.1543
T	20 19 19 19 w=97.6016	(1000)
End page
Finish
